server/src/actors/project.rs

```diff
@@ -334,11 +342,15 @@ impl ProjectState {
 
     /// Returns whether this state is outdated and needs to be refetched.
     pub fn outdated(&self, config: &Config) -> bool {
+        // Jitter is used to compute a random interval between [0; 2 * expiry_interval]
+        let jitter = config.cache_timeout_jitter();
+        let factor = jitter * (self.seed * 2.0);
```

Comment:
TODO: Double check this when less tired. Math is hard. 

`seed` is standard deviated, meaning it centers around 0.5. This makes  `seed * 2` center around 1.0, but jitter will make it center around something smaller. This actually needs to be something more like:

```
1 + jitter * (self.seed * 2.0 - 1)
```